.semaphore/semaphore.yml

```diff
@@ -115,5 +124,10 @@ blocks:
           commands:
             - echo "Setting up build..."
             - meson setup build.clang -Dusewebcam=true -Dusemididrum=true -Dusewebserver=true
+            - artifact push job --expire-in 1w build.clang/meson-info
+            - artifact push job --expire-in 1w build.clang/meson-logs
+            - artifact push job --expire-in 1w build.clang/build.ninja
+            - artifact push job --expire-in 1w build.clang/compile_commands.json
             - echo "Building..."
             - meson compile -C build.clang
+            - artifact push job --expire-in 1w build.clang/.ninja_log
```

Comment:
What are these artifacts? What do they provide? I think they don't have a function outside of the ci? Logs and such are viewable in semaphore

Reply:
These are the build logs.  They provide much more information than what the build system shows to stdout, which is what semaphore shows.  That being said, job artifacts do not appear to be available to non logged in users, so they provide little value as-is.